docs/voice/telephony-session-notifications.md

```diff
@@ -0,0 +1,225 @@
+# Telephony Session Notification
+
+A developer can use Telephony Session Notification to detect all changes in the status of a call effectively. 
+
+## What is Telephony Session Notification?
+
+A Telephony Session Notification (Telephony Session Notification) is a series of notifications or events a developer can use to track an end-to-end telephony call in RingCentral. The call can be from an external PSTN or any mobile number to a RingCentral extension, or it can be from a RingCentral Digital Line to an external PSTN number, or it can be between two RingCentral extensions. The Telephony Session Notification provides detailed end-to-end call sessions and party (endpoints/user) level information to enable developers to take specific actions or build custom reports based on that call information. The Telephony Session Notification events can be subscribed both at an Account level or at an Extension level.
+
+Account Level Telephony Session Notification -  Will track and provide notifications for all the extensions within an account.
+
+Extension Level Telephony Session Notification - Will track calls on a specific extension.
+
+Detailed instructions on the [Notification types](https://developers.ringcentral.com/api-reference/Account-Presence-Event) are available in the API reference.
+
+Detailed instructions on how to subscribe to [RingCentral events](https://developers.ringcentral.com/guide/notifications) or notifications are available in the API reference
+
+
+## What are the use cases?
+
+<h5>Call Analytics</h5> -  Developers can use the raw data from Telephony Session Notification events and build call analytics dashboards replicating live reports which use the same event streams. Some of the standard call analytics metrics and their corresponding formulas are:
+
+    - Ring Time
+    - Call Talk Time
+    - Call Hold Time
+
+!!! note "Development Considerations."
+    The individual event streams needs to be stored in a persistent storage like MySql DB to peform the calculations.
+
+
+### Ring Time - This is the amount of time the call was ringing before it was answered. It is the time duration between Proceeding and Answered state.
+
+
+  Ring Time = Time difference between Proceeding and Answered
+    
+  Example:
+
+  Step 1:  Get the `'eventTime': '2020-06-23T23:46:11.243Z'` for `'code': 'Proceeding'`
+
+
+  Step 2:  Get the `'eventTime': '2020-06-23T23:46:23.451Z'` for `'code': 'Answered'`
+
+
+
+  Step 3:  Ring Time =  Timestamp in Step 2 - Timestamp in Step 1 = `'eventTime': '2020-06-23T23:46:23.451Z'` - `'eventTime': '2020-06-23T23:46:11.243Z'` = 12 seconds
```

Comment:
Please implement Markdown ordered list.

https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax